doc/api/stream.md

```diff
@@ -2501,21 +2501,34 @@ This method calls `fn` on each chunk of the stream in order, passing it the
 result from the calculation on the previous element. It returns a promise for
 the final value of the reduction.
 
-The reducer function iterates the stream element-by-element which means that
-there is no `concurrency` parameter or parallelism. To perform a `reduce`
-concurrently, it can be chained to the [`readable.map`][] method.
-
 If no `initial` value is supplied the first chunk of the stream is used as the
 initial value. If the stream is empty, the promise is rejected with a
 `TypeError` with the `ERR_INVALID_ARGS` code property.
 
 ```mjs
 import { Readable } from 'node:stream';
 
-const ten = await Readable.from([1, 2, 3, 4]).reduce((previous, data) => {
-  return previous + data;
-});
-console.log(ten); // 10
+const totalViews = await Readable.from(videosUrl)
+  .reduce(async (total, url) => {
+    const views = await getViewsAsync(url);
+    return total + views;
+  }, 0);
+
+console.log(totalViews);
```

Comment:
Examples should not use random undeclared things like the `videosUrl`/`getViewsAsync()`, the examples should be snippets that the users can copy-paste and run without errors.

Reply:
thanks, updated